data/pirate jobs.txt

```diff
@@ -0,0 +1,2750 @@
+	job
+	repeat
+	description "The captain of the <npc> will pay you <payment> to escort them and their illegal cargo safely through law enforced space to reach <destination> by <date>."
+	deadline 12
+	to offer
+		random < 30
+		"combat rating" > 600
+		"reputation: Pirate" > 0
+	source
+		government "Pirate"
+		attributes "south pirate"
+	destination
+		government Republic "Free Worlds" Syndicate Neutral
+		attributes south
+		distance 3 6
+	npc
+		system destination
+		personality staying
+		government Militia
+		fleet "Large Militia"
+	npc accompany save
+		government Pirate
+		personality escort
+		fleet
+			names "pirate"
+			variant 2
+				"Modified Argosy"
+			variant 2
+				"Modified Argosy (Heavy)"
+			variant 2
+				"Modified Argosy (Blaster)"
+			variant 1
+				"Bastion"
+			variant 1
+				"Bastion (Laser)"
+			variant 1
+				"Bastion (Heavy)"
+	on complete
+		payment 240000
+		dialog "The captain of the <npc> thanks you for escorting them safely, and pays you <payment>."
+	on visit
+		dialog "You have reached <planet>, but you left the <npc> behind! Better depart and wait for them to arrive in this star system."
+
+mission "Escort Stolen Vessel (Extra Large, South, Dangerous Path)"
+	name "Protect ship to <planet>"
+	job
+	repeat
+	description "The captain of the <npc> will pay you <payment> to escort them and their stolen vessel safely through law enforced space to reach <destination> by <date>."
+	deadline 12
+	to offer
+		random < 15
+		"combat rating" > 1000
+		"reputation: Pirate" > 0
+	source
+		government Republic "Free Worlds" Syndicate Neutral
+		attributes south
+	destination
+		government "Pirate"
+		attributes "south pirate"
+		distance 3 6
+	npc
+		system destination
+		personality staying
+		government Militia
+		fleet "Small Militia"
+	npc
+		system destination
+		personality staying
+		government Militia
+		fleet "Large Militia"
+	npc accompany save
+		government Pirate
+		personality escort
+		fleet
+			names "pirate"
+			variant 2
+				"Bastion"
+				"Blackbird"
+			variant 2
+				"Bastion (Laser)"
+				"Clipper"
+			variant 2
+				"Bastion (Heavy)"
+				"Bounder"
+			variant 1
+				"Falcon"
+				"Hauler"
+			variant 1
+				"Falcon (Laser)"
+				"Freighter"
+			variant 1
+				"Falcon (Heavy)"
+				"Heavy Shuttle"
+	on complete
+		payment 375000
+		dialog "The captain of the <npc> thanks you for escorting them safely, and pays you <payment>."
+	on visit
+		dialog "You have reached <planet>, but you left the <npc> behind! Better depart and wait for them to arrive in this star system."
+
+mission "Elminating Law Enforcement (North, Small) [0]"
+	name "Navy near <system>"
+	repeat
+	job
+	description "A Navy fleet near the <system> system has halted our raids. Destroy them and return to <planet> for a payment of <payment>."
+	to offer
+		random < 25
+		"combat rating" > 500
+	source
+		government "Pirate"
+		attributes "north pirate"
+	npc kill
+		personality heroic staying
+		government "Bounty: Republic"
```

Comment:
Maintaining a separate "Bounty" government for each human faction could get really messy. The only reason a separate "Bounty" government exists at all was to make sure that other ships don't kill your bounty for you (which would happen if it were a "Pirate"). But in this case you actually *want* everyone else to come to the bounty's aid, for the sake of realism, so the bounty should probably just use the government it comes from.